src/components/ShareList.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { shareList, useAuth } from '../api';
+
+export function ShareList() {
+	const [emailData, setEmailData] = useState('');
+
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+
+	const handleTextChange = (event) => {
+		return setEmailData(event.target.value);
+	};
+
+	const shareCurrentList = async () => {
+		const listShared = await shareList(listPath, userId, emailData);
```

Comment:
Breaking change: It seems like `listPath` is not declared or initialized before being passed to the shareList function leading to ![this](https://cdn.zappy.app/3e7a5c1318dc867f80dc572e3778b39d.png) console error!

Reply:
Great catch! I am going to commit next week to writing tests!